### DISCUSSION TITLE
Removing supply drops

### DISCUSSION THREAD

**Discussion by IsProbablyCatto** [TEHE-Studios/ExpandedHelicopterEvents#136] [Q&A]
Hello! While playing near the first 10 days of my playthrough, a helicopter dropped a supply crate infront of the rosewood firestation where I was at. I've looted the contents but cannot find a way to remove it as it blocks some future building I wish to do. Is there any way as disasembling doesn't seem to work, cheers!

**Comment by Chuckleberry-Finn** [comment]
At the moment it is a vehicle object - so it can be towed.
